NotesApp/Scenes/Editor/EditorViewController.swift

```diff
@@ -80,42 +89,31 @@ class EditorViewController: UIViewController {
     private func setupUI() {
         view.backgroundColor = UIColor(hex: "#252525")
         
-        setupCustomBackButton()
-        
         titleTextView.addDoneButtonToolbar()
         bodyTextView.addDoneButtonToolbar(onDone: (target: self, action: #selector(customDoneAction)))
         
         view.addSubview(titleTextView)
         view.addSubview(bodyTextView)
```

Comment:
this is not UI, this is hierarchy

Reply:
I don't quite understand what's the problem here